WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -182,11 +181,16 @@ private extension NewOrderViewModel {
             .assign(to: &$navigationTrailingItem)
     }
 
-    func setInitialOrderStatus() {
-        guard let pendingSiteStatus = currentSiteStatuses.first(where: { $0.status == .pending }) else {
-            return
-        }
-
-        orderStatus = pendingSiteStatus
+    /// Updates status badge viewmodel based on status order property.
+    ///
+    func configureStatusBadgeViewModel() {
```

Comment:
Should we have a unit test to check that the status badge view model is updated as expected?

Reply:
Good catch! Added in f1db99e.